okxchain/chain.json

```diff
@@ -0,0 +1,108 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "okxchain",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "OKC",
```

Comment:
Are we sure we want the chain's pretty name to be "OKC"? This is meant to be how people should say the name of the chain. It is my opinion that a user reading this is unlikely to know what this means, whereas something like "OKEx Chain" would be instantly recognizable.